tools/rack/rack.plugin/src/com/ge/research/rack/UploadNodegroupsHandler.java

```diff
@@ -109,6 +141,10 @@ public static List<String> getNodegroupFilepaths(final String basedir) {
     @Override
     public Object execute(final ExecutionEvent event) throws ExecutionException {
 
+        if (!startRun()) {
+            return null;
```

Comment:
We may inform the user that we are debouncing, that there is another uploadnodegroups in progress